backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -208,7 +234,9 @@ describe('CommunityResolver', () => {
               {
                 id: 5,
                 foreign: foreignCom2.foreign,
-                publicKey: expect.stringMatching('publicKey-ForeignCommunity'),
+                publicKey: expect.stringMatching(
+                  '43c72cb81416121f5eb98affa4fb3360088719e80db6aaa13ff7e74d3f669307',
+                ),
```

Comment:
Why not using the variable `foreignCom2.publicKey` instead of hardcoded string?

Reply:
FederatedCommunity.publicKey should be a Buffer type, graphql returns a hex string.
But you right I could make it way better. Save for future test, if the key will be actually validated.